crates/pop-drink/src/mock.rs

```diff
@@ -0,0 +1,107 @@
+use frame_support::{
```

Comment:
Assume this is internal, only for testing that the macros and functions work correctly?

Reply:
Resolved